server/bin/qpid/README.md

```diff
@@ -1,3 +1,5 @@
+[Deprecated since Candlepin-4.0.0]
+
```

Comment:
No point in adding this here; we can just delete the whole file (the whole `server/bin/qpid/` directory actually). It's still going to be available in older branches.

Reply:
The `server/bin/qpid_watch.py` script also needs removing

Reply:
The ```configure-qpid.sh``` script inside ```server/bin/qpid/``` is being used by ```docker/candlepin-base/cp-test```. Shouldn't  this affect our docker image builds ? 

I removed ```server/bin/qpid_watch.py``` script.

Reply:
You're right. in that case let's keep it!